gsoc/students-guideline.md

```diff
@@ -12,13 +12,13 @@ Get started checking out the [Google Summer of Code Guides for students](https:/
 
 ### How to apply to HSF GSoC
 
-1. Take a look at the list of participating [projects]({{ site.baseurl }}/activities/gsoc.html#projects-in-2022) this year. If there are no projects or just a few, note that more will be added according to this year's [timeline]({{ site.baseurl }}/activities/gsoc.html#timeline).
+1. Take a look at the list of participating [projects]({{ site.baseurl }}/activities/gsoc.html#projects-in-2024) this year. If there are no projects or just a few, note that more will be added according to this year's [timeline]({{ site.baseurl }}/activities/gsoc.html#timeline).
   * There you will find a list of proposals for each project, their descriptions and the contact information of their mentors.
-  * There is also a list with [all the project proposals]({{ site.baseurl }}/gsoc/2023/summary.html) sorted in alphabetical order.
+  * There is also a list with [all the project proposals]({{ site.baseurl }}/gsoc/2024/summary.html) sorted in alphabetical order.
   <!-- Next <br><br>, add an extra new line, otherwise there is no space between point 2 and previous nested bullet  -->
   <br><br>
 
-2. Once you have checked all our proposals and picked the one you like most, please contact the corresponding mentors (their emails can be found at the end of each proposal) sending them your CV and the motivation for picking their proposal. Please do this only after the list of accepted Organizations is published on `March 7`, but not after `March 20`, to give time to mentors to evaluate your technical skills. You are encouraged to apply to a maximum of two project proposals.
+2. Once you have checked all our proposals and picked the one you like most, please contact the corresponding mentors (their emails can be found at the end of each proposal) sending them your CV and the motivation for picking their proposal. Please do this only after the list of accepted Organizations is published on `Feb 21`, but not after `March 17`, to give time to mentors to evaluate your technical skills. You are encouraged to apply to a maximum of two project proposals.
```

Comment:
Minor comment: Font choice is rather odd. How about **bold** or *italic*? This is generally true of this page.